ai_genomics/pipeline/gtr/gtr_utils.py

```diff
@@ -0,0 +1,22 @@
+# Scripts to explore definitions in the GtR data
+import json
+import logging
+import boto3
+from typing import List, Dict
+
+from toolz import pipe
+
+
+def fetch_gtr(table_name: str) -> List[Dict]:
+    """Fetch a json gtr object"""
+    s3 = boto3.resource("s3")
+    ai_genomics_bucket = s3.Bucket("ai-genomics")
+
+    logging.info(f"Fetching table {table_name}")
```

Comment:
does it matter that we are using logging instead of from `ai_genomics import logger`? I'm not super familiar with all the functionality of logging so just curious!!